app/komponenter/confirmationPanel.tsx

```diff
@@ -0,0 +1,38 @@
+import { ConfirmationPanel } from '@navikt/ds-react';
+import { useState } from 'react';
+import { Button } from '@navikt/ds-react';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from './tekstBlokk/tekstBlokk';
+
+interface BekreftelsePanelProp {
+  melding: SanityDokument | undefined;
+  samtykke: SanityDokument | undefined;
+  feilmelding: SanityDokument | undefined;
+}
+
+const BekreftelsePanel: React.FC<BekreftelsePanelProp> = ({
+  melding,
+  samtykke,
+  feilmelding,
+}: BekreftelsePanelProp) => {
+  const [lest, settLest] = useState(false);
+  const [trykkVidere, settTrykkVidere] = useState(false);
+
+  return (
+    <div>
+      <ConfirmationPanel
+        checked={lest}
+        label={<TekstBlokk tekstblokk={samtykke} />}
+        onChange={() => settLest(x => !x)}
+        error={!lest && trykkVidere && <TekstBlokk tekstblokk={feilmelding} />}
```

Comment:
Kan noe av denne logikken trekkes ut i egen variabel `skalViseFeilmelding`?

Reply:
Det tror jeg absolutt! Men dette tenkte vi å justere når vi setter opp logikken for start-knappen, ettersom det uansett vil endre logikken her